app/assets/stylesheets/cclow/layout.scss

```diff
@@ -4,6 +4,10 @@
 
 $search-field-height: 3.75rem;
 
+.header__margin {
+  margin-top: var(--fixed-navbar-height);
```

Comment:
Nice! I have to use those more often

Reply:
yeah! It's really powerful

Reply:
very straightforward article about css variables and how to take advantage of them: https://dev.to/nirazanbasnet/understanding-css-variables-4cj0

Reply:
Thanks!